BraveWallet/Crypto/Stores/NetworkStore.swift

```diff
@@ -70,6 +91,10 @@ public class NetworkStore: ObservableObject {
     _ network: BraveWallet.NetworkInfo,
     completion: @escaping (_ accepted: Bool, _ errMsg: String) -> Void
   ) {
+    guard network.coin == .eth else {
+      completion(false, "Not supported")
```

Comment:
should be show a popup?

Reply:
We do show [an alert on this view for failures](https://github.com/brave/brave-ios/blob/development/BraveWallet/Settings/CustomNetworkDetailsView.swift#L408) already, however we also [hardcode the network coin to ethereum](https://github.com/brave/brave-ios/blob/development/BraveWallet/Settings/CustomNetworkDetailsView.swift#L508), so this guard isn't really needed yet.